schemas/stat_status.fbs

```diff
@@ -0,0 +1,29 @@
+// ATTENTION
+//
+// This schema is still under development and may still break in the future.
+// Backwards compatibility is so far not guaranteed.
+// dominik.werder@gmail.com
+
+
+// Status updates from components in the system.  The actual status information
```

Comment:
I think "service" is a more appropriate term than "component".

Reply:
:+1: